bot.py

```diff
@@ -83,12 +83,15 @@ def mini(white_team: teams.Team, bot_team: teams.Team, depth, max_from_previous_
             except chess_utils.DidntMove:
                 pass
 
+    if best_score == MAXIMUM_SCORE:
```

Comment:
Add comment before and explain what are you doing here.

Reply:
change it and fixed it :lab_coat:  :pager: